### DISCUSSION TITLE
Chromebook Recovery Utility

### DISCUSSION THREAD

**Discussion by gomes-web** [3kh0/ext-remover#1074] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there any way you can use Chromebook Recovery Utility without downloading the app because I don't have a personal computer.

**Comment by gomes-web** [comment]
btw I'm on version 120.0.6099.310

**Reply by Blobby-Boi** [reply, marked as answer]
use skoiovox

**Reply by gomes-web** [reply]
I cant because alt+shift+s got blocked for me

**Reply by Blobby-Boi** [reply]
i thought you are in 120

**Reply by gomes-web** [reply]
I am but it still wont let me do alt+shift+s

**Reply by VFXKLIPS** [reply]
Tell me more because skiovox is blocked for me and I have an andriod

**Reply by SIMONHSCH** [reply]
you can plug in your phone on your ChromeBook

**Reply by VFXKLIPS** [reply]
> you can plug in your phone on your ChromeBook

Then what do I do after when I plug it in
